### PR TITLE
Switch from RCurl to httr.

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
@jiwalker-usgs @jread-usgs @lawinslow 

Could someone have a look at this?

**Comment by jiwalker-usgs** [comment]
closed?

File: R/importRDB1.r [hunk 1]
```diff
@@ -93,10 +99,10 @@ importRDB1 <- function(obs_url, asDateTime=TRUE, convertType = TRUE, tz=""){
                           "America/Phoenix","America/Metlakatla","UTC"))
   }
   
-
   if(file.exists(obs_url)){
     doc <- obs_url
   } else {
+    # doc <- getWebServiceData(obs_url, add_headers(Accept-Encoding = "gzip"))
```
Comment (jiwalker-usgs) [inline R/importRDB1.r]: Cool that this didn't change, probably can remove the commented out code.

Comment (ldecicco-USGS) [inline R/importRDB1.r]: I think the commented-out stuff is not in final release (but I do have to work on being better about not committing that!)

https://github.com/USGS-R/dataRetrieval/blob/master/R/importRDB1.r

**Comment by ldecicco-USGS** [comment]
Yeah, apparently `httr` is going to depend on `xml2` in the very near-term. So, until I can completely switch from `XML` to `xml2`, it's better to stick with `RCurl`.

**Comment by jread-usgs** [comment]
agreed.

**Pull request closed without merging**